test/fixed_vector_test.cpp

```diff
@@ -369,6 +369,17 @@ TEST(FixedVector, Initializer)
     EXPECT_TRUE(are_equal(v2, std::array{66, 55}));
 }
 
+TEST(FixedVector, SizeDeduction)
```

Comment:
For the same reason, maybe `MaxSizeDeduction` ?

Reply:
Done!